src/AspNetCoreModuleV2/CommonLib/GlobalVersionUtility.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "stdafx.h"
+#include <experimental/filesystem>
+
+namespace fs = std::experimental::filesystem;
+
+// throws runtime error if no request handler versions are installed.
+std::wstring
+GlobalVersionUtility::GetGlobalRequestHandlerPath(PCWSTR pwzAspNetCoreFolderPath, PCWSTR pwzHandlerVersion, PCWSTR pwzHandlerName)
+{
```

Comment:
please add debug assert for the input parameters which should not be NULL indicating the contact between caller and callee